src/Makefile

```diff
@@ -4,7 +4,7 @@
 
 #=============== c compiler ==================================
 #CC = g++
-CC = g++-4.8
+CC = g++
```

Comment:
Make it `CXX ?= g++`  of course that's a lot of changes thereafter, so may be another PR for the change to CXX but at least put `?=` so it will pick up the environment.